liveview-android/src/main/java/org/phoenixframework/liveview/data/dto/BadgeBoxDTO.kt

```diff
@@ -0,0 +1,130 @@
+package org.phoenixframework.liveview.data.dto
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.material3.Badge
+import androidx.compose.material3.BadgeDefaults
+import androidx.compose.material3.BadgedBox
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.contentColorFor
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.graphics.Color
+import org.phoenixframework.liveview.data.core.CoreAttribute
+import org.phoenixframework.liveview.domain.base.ComposableBuilder
+import org.phoenixframework.liveview.domain.base.ComposableView
+import org.phoenixframework.liveview.domain.base.ComposableViewFactory
+import org.phoenixframework.liveview.domain.base.PushEvent
+import org.phoenixframework.liveview.domain.extensions.toColor
+import org.phoenixframework.liveview.domain.factory.ComposableTreeNode
+import org.phoenixframework.liveview.ui.phx_components.PhxLiveView
+
+/**
+ * Material Design badge box.
+ * It must be at most two children: a `Badge` and the content component, that can be any component.
+ * Usually, the content is an `Icon` like below.
+ * ```
+ * <BadgeBox>
```

Comment:
Is this `BadgeBox` or `BadgedBox`? The `Compose` function uses `BadgedBox`.

Reply:
My mistake. I will fix that.